### PR TITLE
Remove mainpage.php

### PR DESCRIPTION
**Pull request by Kamahl19**
Do not forget to include deletion to `/install_instructions/update_*.xml` file!

### CODE REVIEW

File: install_instructions/update_from_0.7.2_to_0.7.3.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	<action-group>
+		<copy>
+			<file from="root/*.*" to="*.*"/>
+		</copy>
+		<delete>
+			<file name="socialnet/js/jquery.floatobject-1.4.js"/>
+			<file name="socialnet/js/jquery.textarea-expander.js"/>
+			<file name="mainpage.php"/>
+		</delete>
+		<php-installer><![CDATA[socialnet/install_sn.php]]></php-installer>
```
Comment (senky) [inline install_instructions/update_from_0.7.2_to_0.7.3.xml:62]: Please, include DB update to this PR, too.

Comment (Kamahl19) [inline install_instructions/update_from_0.7.2_to_0.7.3.xml:62]: I dont think so. SQL update is not related with this PR. We will add sql update to the PR before releasing 0.7.3

**Comment by senky** [comment]
Do this:
1. `git reset --hard 91eae3a`
2. `git fetch upstream`
3. `git rebase upstream/develop`
4. resolve the merging conflicts
5. https://github.com/Kamahl19/phpBB-Social-Network/commit/91eae3a199ab24f0f2426f5bc8c1bf2c9536973f#root-language-en-mods-socialnet_acp-php-P5
6. git add and git commit
7. `git push -f origin i/76`

**Comment by Kamahl19** [comment]
thanks, how can i resolve merge conflict? do you an idea why this happened ?

**Comment by senky** [comment]
`git mergetool`

Conflicts happen, when you edit lines, which have been modified by another PR, which was closed earlier than this one.

**Pull request closed without merging**